src/views/AgmExtension.vue

```diff
@@ -63,6 +63,10 @@
             <AgmExtensionEvaluation
               class="mt-8"
               :data.sync="data"
+              :isEligible="false"
```

Comment:
You don't need to pass this in -- it's already part of the data object on line 65.

Reply:
Removed, thanks.